src/dev/storybook/aliases.ts

```diff
@@ -31,6 +31,7 @@ export const storybookAliases = {
   expression_reveal_image: 'src/plugins/expression_reveal_image/.storybook',
   expression_shape: 'src/plugins/expression_shape/.storybook',
   expression_tagcloud: 'src/plugins/chart_expressions/expression_tagcloud/.storybook',
+  files: 'x-pack/plugins/files/.storybook',
```

Comment:
Sorry about this, the buildkite version that needs to be updated is at `.buildkite/scripts/steps/storybooks/build_and_upload.ts`.  This is legacy back from the jenkins pipeline.